at_client/src/main/java/org/atsign/client/util/RegisterUtil.java

```diff
@@ -110,33 +178,37 @@ public Boolean registerAtsign(String email, AtSign atsign, String registrarUrl,
+     *                     with previously received OTP. The second follow-up call
      *                     is automated by this client using new atsign for user
      *                     simplicity
      * @return Case 1("verified") - the API has registered the atsign to
      *         provided email and CRAM key present in HTTP_RESPONSE Body.
      *         Case 2("follow-up"): User already has existing atsigns and new atsign
-     *         registered successfully. To receive the CRAM key, follow-up by calling
-     *         the API with one of the existing listed atsigns, with confirmation set to true.
+     *         registered successfully. To receive the CRAM key, follow-up by
+     *         calling
+     *         the API with one of the existing listed atsigns, with confirmation
+     *         set to true.
      *         Case 3("retry"): Incorrect OTP send request again with correct OTP.
-     * @throws IOException           thrown if anything goes wrong while using the HttpsURLConnection.
-     * @throws AtException           Case 1: If user has exhausted 10 free atsign
-     *                               quota
-     *                               Case 2: If API response is anything other than
-     *                               HTTP_OK/Status_200.
+     * @throws IOException thrown if anything goes wrong while using the
+     *                     HttpsURLConnection.
+     * @throws AtException Case 1: If user has exhausted 10 free atsign
+     *                     quota
+     *                     Case 2: If API response is anything other than
+     *                     HTTP_OK/Status_200.
      */
-    public String validateOtp(String email, AtSign atsign, String otp, String registrarUrl, String apiKey,
+    public static String validateOtp(String email, AtSign atsign, String otp, String registrarUrl, String apiKey,
             Boolean confirmation)
             throws IOException, AtException {
         Map<String, String> paramsMap = Stream.of(
-                        new SimpleEntry<>("atsign", atsign.withoutPrefix()),
-                        new SimpleEntry<>("email", email),
-                        new SimpleEntry<>("otp", otp),
-                        new SimpleEntry<>("confirmation", confirmation.toString()))
+                new SimpleEntry<>("atsign", atsign.withoutPrefix()),
+                new SimpleEntry<>("email", email),
+                new SimpleEntry<>("otp", otp),
+                new SimpleEntry<>("confirmation", confirmation.toString()))
                 .collect(toMap(SimpleEntry::getKey, SimpleEntry::getValue));
         String paramsJson = objectMapper.writeValueAsString(paramsMap);
 
-        HttpsURLConnection httpsConnection = postRequestToAPI(new URL(registrarUrl + Constants.VALIDATE_OTP), apiKey, paramsJson);
+        HttpsURLConnection httpsConnection = postRequestToAPI(new URL(registrarUrl + Constants.VALIDATE_OTP), apiKey,
```

Comment:
There are several diffs in this PR due to format changes. Please can you set your hard line wrap to 150? It makes PRs much easier to review if they are not full of format changes. Similarly with the line-level annotations e.g. `@SuppressWarnings("unchecked")` ... my preference generally is to keep them on the same line. Your preference may differ but again it makes it more difficult to review the PR if it's a mixture of 'real' changes and format changes. If you don't like the formatting, generally the best practice is to initially create a PR which **only** changes the formatting. And then follow up with a PR for the 'real' changes.

Reply:
The formatting changes were automatically performed by the IDE. I have set the wrap to 150 now. I tried to revert most of the formatting changes but let some changes be in the code which were needed. I'll make sure to separate formatting and actual changes from the upcoming PR.